keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueOutputFormat.java

```diff
@@ -48,32 +46,48 @@
  */
 @PublicApi
 @ThreadSafe
-public class KeyValueOutputFormat extends FileOutputFormat<BytesWritable, BytesWritable> {
+public final class KeyValueOutputFormat extends FileOutputFormat<BytesWritable, BytesWritable> {
+  /**
+   * @param conf MapReduce job configuration
+   * @return the task's temporary output path ${mapred.out.dir}/_temporary/_${taskid}
+   */
+  public static TachyonURI getTaskOutputURI(JobConf conf) {
+    return getJobOutputURI(conf).join(FileOutputCommitter.TEMP_DIR_NAME).join("_"
+        + TaskAttemptID.forName(conf.get("mapred.task.id")).toString());
+  }
+
+  /**
+   * @param conf MapReduce job configuration
+   * @return the job's output path
+   */
+  public static TachyonURI getJobOutputURI(JobConf conf) {
+    return new TachyonURI(new TachyonURI(FileOutputFormat.getOutputPath(conf).toString())
```

Comment:
fixed, we drop scheme and authority in key-value master client now, just like in filesystem master client.